mlir/utils/jenkins/Jenkinsfile

```diff
@@ -1057,5 +1052,68 @@ pipeline {
                 }
             }
         }
+
+        stage ("Code coverage") {
+            when {
+                beforeAgent true;
+                equals expected: true, actual: params.runCodeCoverage;
+            }
+            matrix {
+                axes {
+                    axis {
+                        name 'CPATH'
```

Comment:
Decision from meeting on 2024-01-17:  coverage variations will be by codepath more than by chip, and for the moment just do mfma because that accounts for most things.

CPATH instead of CODEPATH because of a bizarre problem in which the longer variable isn't expanded underneath the withCredentials.  No reason for that, but I didn't dig into it further than proving that it exists.